dockstore-webservice/src/main/resources/swagger.yaml

```diff
@@ -6354,6 +6377,71 @@ definitions:
           format: "int64"
           readOnly: true
       description: "This describes one workflow in the dockstore"
+  Category:
+    type: "object"
+    required:
+    - "name"
+    properties:
```

Comment:
can you make a code comment in the category.java saying which of these properties do not apply?